modules/openapi-generator/src/main/resources/dart2/serialization/json_serializable/json_serializable_class.mustache

```diff
@@ -49,10 +49,7 @@ class {{{classname}}} {
   Map<String, dynamic> toJson() => _${{{classname}}}ToJson(this);
 
   @override
-  String toString() {
-    return toJson().toString();
-  }
-
+  String toString() => '${toJson()}';
```

Comment:
same here as my comment above, calling `.toString()` is more readable than interpolation

Reply:
Same answer as above :)